internal/wasm/jit/engine.go

```diff
@@ -85,9 +84,9 @@ type (
 		callFrameStackPointer uint64
 	}
 
-	// moduleContext holds the per-function call specific module information.
+	// callContext holds the per-function call specific module information.
 	// This is subject to be manipulated from JITed native code whenever we make function calls.
-	moduleContext struct {
+	callContext struct {
```

Comment:
oh well this is not exactly callContex... moduleContext holds per-module instance specific information, and change whenever enters/exits  new functions.. so I would revert this

Reply:
and this is nothing to do with `wasm.ModuleContext` or `wasm.CallCotext` (sorry for confusing)

Reply:
whoops. refactor tool fail :D will revert!

Reply:
ok rolled that back. thanks for noticing